helm-go-lib/main.go

```diff
@@ -181,10 +182,30 @@ func buildActionConfig(namespace string, kubeToken string, apiServer string, caF
 	kubeConfig.CAFile = &caFile
 	kubeConfig.Namespace = &namespace
 
+	customConfig := &CustomConfigFlags{kubeConfig}
+
 	actionConfig = new(action.Configuration)
-	if err := actionConfig.Init(kubeConfig, namespace, os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
+	if err := actionConfig.Init(customConfig, namespace, os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
 		return nil, err
 	}
 
 	return actionConfig, nil
 }
+
+// Override ToRESTConfig to set QPS and Burst
+type CustomConfigFlags struct {
```

Comment:
This is the crux: I am overridding the `ConfigFlags` struct (which implements the `RESTClientGetter` interface) to be able to set `QPS`/`Burst` on the `RESTConfig`. On a fiab I see the log statement, so I think it's working. (Let me know if there's a more Go-idiomatic way to do this.)